tasks/tests.py

```diff
@@ -61,7 +71,8 @@ def setUp(self):
         self.task2 = Task.objects.create(
             text="Sleep", project=self.project1, completed=False
         )
-
+        self.user = self.create_user_and_return_user()
+        self.client.login(username=self.username, password=self.password)
```

Comment:
You should not make login API request in all test cases, this would slow down test case. Instead use `login` method of django.